components/formatted_markdown_message.jsx

```diff
@@ -17,12 +19,18 @@ export class CustomRenderer extends marked.Renderer {
     }
 
     link(href, title, text) {
+        const siteURL = getSiteURL();
+        const openInNewTab = shouldOpenInNewTab(href, siteURL);
+
         if (this.disableLinks) {
             return text;
         }
         if (href[0] === TARGET_BLANK_URL_PREFIX) {
             return `<a href="${href.substring(1, href.length)}" rel="noopener noreferrer" target="_blank">${text}</a>`;
         }
+        if (openInNewTab) {
+            return `<a href="${href}" target="_blank">${text}</a>`;
+        }
```

Comment:
I'm thinking we should add the same `rel="noopener noreferrer"` attribute for this new addition.

Reply:
After thinking about this, it makes sense, thanks!